### PR TITLE
feat(intellij): enable jumping to target & configuration from toolwindow

### LINKED ISSUE #1725: [intellij] Jump To Source hotkey should navigate to the needed section in JSON
## Description

#1678 can be further improved by jumping to the corresponding **section** in the JSON file

## Motivation

This is what JetBrains does for NPM Scripts, and not only

### PR DESCRIPTION
**Pull request by MaxKless**
fixes #1725

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -14,5 +14,25 @@
...
+    "statusBar.foreground": "#e7e7e7",
+    "statusBarItem.hoverBackground": "#2f7c47",
+    "statusBarItem.remoteBackground": "#215732",
+    "statusBarItem.remoteForeground": "#e7e7e7",
+    "titleBar.activeBackground": "#215732",
+    "titleBar.activeForeground": "#e7e7e7",
+    "titleBar.inactiveBackground": "#21573299",
+    "titleBar.inactiveForeground": "#e7e7e799"
+  },
+  "peacock.color": "#215732"
```
Comment (Cammisuli) [inline .vscode/settings.json:37]: revert this

**Review by Cammisuli — DISMISSED** [review]
Just revert the color changes in the vscode settings, and it should be good to go

**Review by Cammisuli — CHANGES_REQUESTED** [review]
see previous comment

File: .vscode/settings.json (hunk 1 above)
Comment (MaxKless) [reply .vscode/settings.json]: oops, reverted

**Pull request merged**